catalog/spatial/registry/registry-source-configuration-handler/src/main/java/org/codice/ddf/registry/source/configuration/SourceConfigurationHandler.java

```diff
@@ -581,9 +581,9 @@ private String getBindingTypeToActivate(List<ServiceBindingType> bindingTypes) {
   }
 
   @SuppressWarnings("squid:S1149" /* Confined by underlying contract */)
```

Comment:
❓ Does this suppression still apply? If not, it should be removed so it doesn't inadvertently mask a real problem down the road.